### ISSUE TITLE
Default values

### ISSUE THREAD

**Issue by antonkulaga** [sjrd/scala-js-ts-importer#15]
When I import ts files I often get methods like def this(`object`: Object3D, size: Double = ???, hex: Double = ???, linewidth: Double = ???) = this()
with a lot of default arguments and corresponding " multiple overloaded alternatives of method "
Maybe for such kind of methods it would be better to generate several methods without defaults, like
  def this(`object`: Object3D) = this()
  def this(`object`: Object3D, size: Double) = this()
  def this(`object`: Object3D, size: Double, hex: Double) = this()
  def this(`object`: Object3D, size: Double, hex: Double, linewidth: Double) = this()

I understand it is verbose but it allows to avoid doing many manual corrections.

**Comment by onilton** [comment]
I don't know if this is the same issue that I'm thinking, but if it is not, I can open another issue.

What about

```
import scala.scalajs.js.UndefOr
import scala.scalajs.js.undefined

def this(object: Object3D, size: UndefOr[Double] = undefined, hex: UndefOr[Double] = undefined, linewidth: UndefOr[Double] = undefined) = this()
```

Could this be a possible solution?

**Comment by sjrd** [comment]
@onilton I don't think this is the same issue, no. The current issue is specifically about the "multiple overloaded alternatives have default arguments" compile error.
